src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -121,7 +121,12 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration, pollIntervalJit
 
     lookupShouldNotify(notification) flatMap { shouldNotify =>
       if (shouldNotify) {
-        sendGridDAO.sendNotifications(List(toThurloeNotification(notification))).map(_.headOption)
+        sendGridDAO.sendNotifications(List(toThurloeNotification(notification))).map(_.headOption) recoverWith {
+          case e: KeyNotFoundException =>
+            logger.info(s"Unable to send notification due to missing key: ${e.getMessage}")
+            Future.successful(None)
```

Comment:
returning None here will have the same effect as the user having turned off notifications: we will ack the pubsub message so we don't attempt to send it again

the lack of the ack is what caused the log spam